main.py

```diff
@@ -21,39 +21,41 @@ async def main():
     url =data['prometheus_url']
     await forecast(metric_list,url)
         
-async def predict_every(metric_name,start_time,end_time,url,prom_query,write_back_metric,forecast_every,forecast_basedon):
+async def predict_every(metric_name,start_time,end_time,url,prom_query,write_back_metric,forecast_every,forecast_basedon,models):
```

Comment:
@WeepingClown13 please make the argument name model instead of models since there will be only 1 model object per metric